spec/models/hmpps_api/offender_spec.rb

```diff
@@ -168,7 +168,7 @@
 
       include_examples 'expect fields to be', nil, [
         :tier, :case_allocation, :mappa_level, :parole_review_date,
-        :welsh_offender, :ldu, :team, :allocated_com_name
+        :welsh_offender, :ldu_name, :team_name, :allocated_com_name
```

Comment:
Same here – we should add `#ldu_email_address` to prove that it's `nil` when not available.